### PR TITLE
[3.3.5] Scripts/ICC: Sindragosas Icetombs deal damage to all other icetombs too

### PR DESCRIPTION
**Pull request by P-Kito**
**Changes proposed**:
-  Icetombs will now damage other icetombs, so the retail tactics of staying not too crowded is now a thing

**Target branch(es)**: 335/6x

**Issues addressed**: Fixes https://github.com/TrinityCore/TrinityCore/issues/7808

**Tests performed**: I have been running that for a longer time now

### CODE REVIEW

**Comment by Aokromes** [comment]
This branch has conflicts that must be resolved
 Use the command line to resolve conflicts before continuing.

**Comment by Keader** [comment]
Working for me :+1:

File: src/server/scripts/Northrend/IcecrownCitadel/boss_sindragosa.cpp [hunk 1]
```diff
@@ -1297,46 +1298,42 @@ class spell_sindragosa_ice_tomb : public SpellScriptLoader
                 return true;
             }
 
-            void SummonTomb()
+            void PeriodicTick(AuraEffect const* aurEff)
```
Comment (Treeston) [inline src/server/scripts/Northrend/IcecrownCitadel/boss_sindragosa.cpp]: Add a check in this method to only summon the block once the player stops moving. similar to the Sapphiron changes in #16524.

**Comment by Shauren** [comment]
Merged in 033f4cddeabe7c0a2ae22257c3c536751d35b92c

**Pull request closed without merging**